src/consumers.ts

```diff
@@ -49,6 +52,139 @@ interface IPipeline<T> {
 /* Diagnostics producer type */
 type DiagnosticProducer = IProducer<Diagnostic[]>;
 
+/* Package vulnerability data */
+class Package
```

Comment:
This should be called as Vulnerability rather than Package.

Reply:
Actually its package's vulnerability data. I feel main object is Package along with vulnerability, advisory, exploit, and recommended fix information. So Package name makes more sense for this group of data.

Reply:
Isn't just a typescript class to represent the CA api response payload which denotes Vulnerability?

Reply:
This is done.